composer.json

```diff
@@ -24,7 +24,7 @@
         }
     ],
     "require": {
-        "php": ">=7.2.5",
+        "php": ">=8.0.2",
         "symfony/polyfill-php80": "^1.22",
```

Comment:
this polyfill is now useless as PHP 8.0 is required anyway

Reply:
removed